spec/lib/msf/base/serializer/readable_text_spec.rb

```diff
@@ -0,0 +1,52 @@
+require 'spec_helper'
+require 'msf/base/serializer/readable_text'
+# require 'rex/post/meterpreter/extensions/stdapi/net/interface'
+# require 'rex/post/meterpreter/extensions/stdapi/net/route'
+
+describe Msf::Serializer::ReadableText do
+
+  let (:subject) {
+    described_class
+  }
+  it "should format text bold" do
```

Comment:
All of these should be inside a context block named after the method under test.  e.g.:
```ruby
context '.format_text' do
  ...
end
```

The convention is `.name` for class methods (as is the case for you here) and `#name` for instance methods.